.github/workflows/.pipeline.yml

```diff
@@ -0,0 +1,27 @@
+name: Lint Styles
+on:
+  push:
+    branches: [ dev, main ]
+    tags:
+      - '*.*.*'
+  pull_request:
+    branches: [ dev ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '14' ]
```

Comment:
I noticed that some of the other projects are using Node 10 in their package.json files (`engines.node`). It's probably a good idea to standardize this, like you have for PHP, so that all developers and CI tools are running lint/build scripts on the same Node version. 14 is the current LTS and probably a good choice. Curious what @arzola and others think.